GDJS/Runtime/gd.ts

```diff
@@ -558,8 +558,30 @@ namespace gdjs {
       return diff / Math.min(absA + absB, Number.MAX_VALUE) < epsilon;
     }
   };
+
+  const asynchronouslyLoadingLibraryPromises: Array<Promise<any>> = [];
+
+  /**
+   * Register a promise that a library will be loaded.
+   *
+   * This method must be called by any library that loads asynchronously.
+   */
+  export const registerAsynchronouslyLoadingLibraryPromise = (
+    promise: Promise<any>
+  ): void => {
+    asynchronouslyLoadingLibraryPromises.push(promise);
+  };
+
+  /**
+   * @returns a promise that all libraries will be loaded.
```

Comment:
```suggestion
   * @returns a promise resolved when all all third party libraries, which need to be loaded before the game startup, are loaded. If a library fails loading, this will be rejected.
```